packages/office-ui-fabric-react/src/components/Persona/PersonaCoin.tsx

```diff
@@ -143,6 +134,44 @@ export class PersonaCoin extends React.Component<IPersonaProps, IPersonaState> {
     );
   }
 
+  @autobind
+  private _onRenderCoin(props: IPersonaProps, size: number): JSX.Element {
+    if (props.onRenderCoin) {
+      console.log(size);
+      return(
+        <div>
```

Comment:
Does the Coin need to be wrapped in `div`s?

Reply:
I think the change I just committed is a bit cleaner.  The div can be removed if a null return from _onRenderCoin is allowed which would just print nothing.